clang/test/compiler-course/yasakova_t_lab1/yasakova_t_user_data_type_test.cpp

```diff
@@ -0,0 +1,135 @@
+// RUN: %clang_cc1 -load %llvmshlibdir/PrintData_Yasakova_Tatiana_FIIT1_ClangAST%pluginext -plugin PrintDataPlugin %s -fsyntax-only 2>&1 | FileCheck %s
+
+// CHECK: A(struct)
+struct A {};
+
+// CHECK: B(struct)
+// CHECK-NOT: |_Fields
+// CHECK: A(struct)
+struct B {
+  struct A {};
+};
```

Comment:
I expected this output
```suggestion
// CHECK: A(struct)
struct A {};

// CHECK: B(struct)
// CHECK-NOT: |_Fields
// CHECK: B::A(struct)
struct B {
  struct A {};
};
```